build/common.sh

```diff
@@ -92,8 +92,8 @@ readonly KUBE_CONTAINER_RSYNC_PORT=8730
 # $1 - server architecture
 kube::build::get_docker_wrapped_binaries() {
   local arch=$1
-  local debian_base_version=v1.0.1
-  local debian_iptables_version=v11.0.3
+  local debian_base_version=v2.1.0
+  local debian_iptables_version=v12.1.0
```

Comment:
We are bumping a major version on 1.16.. will that make a difference here?

Reply:
@cblecker -- Should be fine, but I can run a mock stage after this merges to validate. I think this was more so the bump not getting back to the 1.16 branch in whichever v2.0.0/v12.0.0 update vs a specific reason the base image shouldn't be kept up-to-date.

Reply:
I'd like to actually inspect the delta.

one thing is that we're backporting the iptables / iptables nft "fix", which we previously suggested we wouldn't,